src/components/spreadsheet/utils/config-tables.js

```diff
@@ -126,6 +126,35 @@ const defaultEnumFilterConfig = {
     isEnum: true,
 };
 
+/**
+ * Default configuration for a boolean filter
+ */
+const defaultBooleanFilterConfig = {
+    filter: 'agTextColumnFilter',
+    agGridFilterParams: {
+        filterOptions: [
+            {
+                displayKey: 'customInRange',
```

Comment:
Choose a better descriptive displayKey and displayKey like 'booleanMatches' for example